src/components/ImageUpload/index.jsx

```diff
@@ -10,22 +10,39 @@ class ImageUpload extends React.Component {
     super(props);
     this.state = {
       value: props.input.value,
+      sizeValidationError: false,
     };
 
     this.handleFilePicked = this.handleFilePicked.bind(this);
-  }
-
-  updateValue(value) {
-    this.setState({ value });
-    this.props.input.onChange(value);
+    this.sizeValidator = this.sizeValidator.bind(this);
   }
 
   handleFilePicked(event) {
     const reader = new FileReader();
+    const file = event.target.files[0];
+    // File size is given in bytes
+    const oneMegabyte = 1048576;
```

Comment:
Thoughts on using the 1024^2 or 1,000,000 definition of Megabyte?

Reply:
As discussed in person, use 1,000,000 definition.

Reply:
Switched to 1,000,000. Also this is a completely arbitrary number. Let me know if we want to actually figure out what size we allow.